src/mixins/view.js

```diff
@@ -119,24 +116,23 @@ export default {
   // Overriding Backbone.View's `delegateEvents` to handle
   // `events` and `triggers`
   delegateEvents: function(eventsArg) {
-    // proxy behavior $el to the view's $el.
-    _.invoke(this._behaviors, 'proxyViewProperties', this);
-
-    var events = _getValue(eventsArg || this.events, this);
 
-    // normalize ui keys
-    events = this.normalizeUIKeys(events);
-    if (typeof eventsArg === 'undefined') {this.events = events;}
+    this._proxyBehaviorViewProperties();
 
-    var combinedEvents = {};
+    var events = this.normalizeUIKeys(
+      _getValue(eventsArg || this.events, this)
+    );
 
-    // look up if this view has behavior events
-    var behaviorEvents = _.result(this, 'behaviorEvents') || {};
-    var triggers = this.configureTriggers();
-    var behaviorTriggers = _.result(this, 'behaviorTriggers') || {};
+    if (typeof eventsArg === 'undefined') {
+      this.events = events;
+    }
 
-    // behavior events will be overriden by view events and or triggers
-    _.extend(combinedEvents, behaviorEvents, events, triggers, behaviorTriggers);
+    var combinedEvents = _.extend({},
```

Comment:
I know this isn't necessarily part of this PR, but I do think this._getBehaviorTriggers() should come before this.configureTriggers(), to match events..  I think behavior triggers were added later and the order wasn't considered.